dev-libs/libarcus/files/libarcus-4.9.1-deprecated-protobuf-calls.patch

```diff
@@ -0,0 +1,30 @@
+--- a/src/Socket_p.h
```

Comment:
My reasoning was that I expect 4.8.0 to disappear soon because of no Python 3.9 in the ebuilds, yet I didn't want to touch the ebuild contents, but okay, I'll adapt it.

Reply:
@expeditioneer is it okay if I rename the file to be versionless and update 4.8.0 also without a revbump?

Reply:
@MathyV i would like to have the version number when this patch was first introduced so it is IMHO better to keep track when this patch was first introduced. This is not a strong opinion i could also be happy with a versionless patch.